subworkflows/nf-core/fasta_binning_concoct/meta.yml

```diff
@@ -60,16 +55,12 @@ output:
       description: |
         Structure: [ val(meta), path(csv) ]
         CSV file containing final cluster assignments of original input contigs
-  - bin:
-      type: file
-      description: |
-        Structure: [ val(meta), path(fasta) ]
-        FASTA files containing CONCOCT predicted bin clusters, named numerically
-        by CONCOCT cluster ID
   - versions:
       type: file
       description: File containing software versions
       pattern: "versions.yml"
+  - bins:
+      description: ""
```

Comment:
I don't really understand this change, why has it deleted the old description and just moved it down?

And why is it removing meta, (which should be in all output channels except versions in this case)

@mirpedrol can you provide more context what tools is trying to do here?

Reply:
The lint command with `--fix` is fixing the structure of the meta, adding all input and output channels, for example in this case `bin` is not a channel, because it's called `bins`. But the description or structure can't be automatized, this must be fixed manually by copying the old descirption and/or checking the `main.nf` of the subworkflow and `meta.yml` of the included modules to know the proper content of the channels.

Reply:
Ahhh, completely missed that typo, my bad!

OK thank you :)